lib/instrumentation/decorator.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { instrument } from './decorator';
+
+//TODO find better way to test this e2w
```

Comment:
```suggestion
//TODO find better way to test this e2e
```
?

Reply:
I hoped to find maybe a better way to test it, but I will remove the TODO.